test/cucumber/Conferencia.feature

```diff
@@ -144,4 +144,26 @@ Feature: conferencia
     And I select the option Serach for Conference at the conference page
     Then a list of all conferences containing that date will be presented in the conference screen
 
+  Scenario: Suggest name of author when I am filling in the corresponding field on web
+    Given I am signing up a new article
+    And the last three authors registered in the system were "Paulo Borba", "Marcelo Dias", "Paulo Viana"
+    When I fill the name of Author field with "Paulo"
+    Then Appears suggestions of names with the names "Paul Borba", "Paulo Viana"
+
+  Scenario: Suggest name of author when I am filling in the corresponding field on web
+    Given I am signing up a new article
+    And the last three authors registered in the system were "Paulo Borba", "Marcelo Dias", "Paulo Viana"    
+    When I fill the Author's name field with "João"
+    Then Not appear suggestions
+
+  Scenario: Suggest name of author when I am filling in the corresponding field
+    Given the system suggested the authors names "Paul Borba", "Paulo Henrique", "Paulo Viana"
```

Comment:
the system has the authors names "Paul Borba", "Paulo Henrique", and "Paulo Viana" in the suggestion's cache